### PR TITLE
Add definitions for CHeliSAInterface

### PR DESCRIPTION
**Pull request by TracerDS**

### CODE REVIEW

File: Client/game_sa/CHeliSA.h [hunk 1]
```diff
@@ -15,10 +15,48 @@
...
+    std::uint32_t field_A14;
 };
+static_assert(sizeof(CHeliSAInterface) == 0xA18, "Invalid size for CHeliSAInterface");
 
 class CHeliSA final : public virtual CHeli, public virtual CAutomobileSA
 {
 public:
     CHeliSA(CHeliSAInterface* pInterface);
+    CHeliSAInterface*       GetHeliInterface() noexcept { return reinterpret_cast<CHeliSAInterface*>(GetInterface()); }
+    //const CHeliSAInterface* GetHeliInterface() const noexcept { return reinterpret_cast<const CHeliSAInterface*>(GetInterface()); }
```
Comment (tederis) [inline Client/game_sa/CHeliSA.h]: Commented out?

Comment (TracerDS) [reply Client/game_sa/CHeliSA.h]: Yes, I couldnt make a const pointer due to some inconsistencies in the parent classes

Comment (tederis) [reply Client/game_sa/CHeliSA.h]: Is this a draft? Because the repository isn't the right place for this kind of comments.

Comment (TracerDS) [reply Client/game_sa/CHeliSA.h]: No, its not a draft. Repo has a lot of these comments. One doesnt make a difference.
In the refactoring PR I will uncomment that and implement const reference.

Comment (tederis) [reply Client/game_sa/CHeliSA.h]: Like I said the repo is not the place for this type of comments. So please, remove it.

**Review by tederis — APPROVED** [review]

**Pull request merged**